### PR TITLE
created test

### PR DESCRIPTION
**Pull request by Sofasmile**

### CODE REVIEW

File: src/main/java/core/basesyntax/SqrtOperation.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+/**
+ * Реализуйте метод sqrt(), вычисляющий квадратный корень числа. В отличие от Math.sqrt(),
+ * это метод при передаче отрицательного параметра должен бросать исключение
+ * java.lang.IllegalArgumentException с сообщением "Expected non-negative number, got ?",
```
[the remark this answers is not in the document]
Comment (Sofasmile) [reply src/main/java/core/basesyntax/SqrtOperation.java]: Illegal

File: src/main/java/core/basesyntax/SqrtOperation.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+/**
+ * Реализуйте метод sqrt(), вычисляющий квадратный корень числа. В отличие от Math.sqrt(),
```
[the remark this answers is not in the document]
Comment (boroda4436) [reply src/main/java/core/basesyntax/SqrtOperation.java]: < p > and < / p > tags

[the remark this answers is not in the document]
Comment (Sofasmile) [reply src/main/java/core/basesyntax/SqrtOperation.java]: done

File: src/main/java/core/basesyntax/SqrtOperation.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+/**
+ * <p>Реализуйте метод sqrt(), вычисляющий квадратный корень числа. В отличие от Math.sqrt(),
+ * это метод при передаче отрицательного параметра должен бросать исключение
+ * java.lang.IllegalArgumentException с сообщением "Expected non-negative number, got ?",
+ * где вместо вопросика будет подставлено фактически переданное в метод число.
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/SqrtOperation.java]:
```suggestion
 * где вместо вопросика будет подставлено фактически переданное в метод число.</p>
```

**Pull request merged**